webui/react/src/pages/ExperimentDetails/Header/ExperimentHeaderLeft.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { ExperimentBase } from 'types';
+
+import css from './ExperimentHeaderLeft.module.scss';
+import ExperimentState from './ExperimentHeaderState';
+
+interface Props {
+  experiment: ExperimentBase;
+}
+
+const ExperimentHeaderLeft: React.FC<Props> = ({ experiment }: Props) => {
```

Comment:
@hkang1 we might consider including this directly in `webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx`